cmd/csi-snapshotter/main.go

```diff
@@ -116,6 +127,19 @@ func main() {
 
 	factory := informers.NewSharedInformerFactory(snapClient, *resyncPeriod)
 	coreFactory := coreinformers.NewSharedInformerFactory(kubeClient, *resyncPeriod)
+	var snapshotContentfactory informers.SharedInformerFactory
+	if *enableNodeDeployment {
+		node := os.Getenv("NODE_NAME")
```

Comment:
It's KUBE_NODE_NAME, not NODE_NAME?

Reply:
I think this is a little confusing.
"KUBE_NODE_NAME" is used to populate the `nodeid` parameter for the CSI hostpath driver:
https://github.com/kubernetes-csi/csi-driver-host-path/blob/v1.7.3/cmd/hostpathplugin/main.go#L50
"NODE_NAME" is used for csi-snapshotter and it is consistent with what is used in external-provisioner:
https://github.com/kubernetes-csi/external-provisioner/blob/v3.0.0/cmd/csi-provisioner/csi-provisioner.go#L129
They should have the same value though, so we don't need two env variables.

It's fine to submit another PR to address this.  @zhucan Can you open an issue to rename "KUBE_NODE_NAME"  to "NODE_NAME" in setup-csi-snapshotter.yaml?  If you have time, please feel free to fix it.  Thanks.

Reply:
@xing-yang ok